src/cli/serve/serve.js

```diff
@@ -141,6 +144,14 @@ module.exports = function (program) {
       process.exit(1); // eslint-disable-line no-process-exit
     }
 
+    process.on('SIGHUP', function reloadConfig() {
+      console.log('Reloading config!');
```

Comment:
I'm worried about using console.log here. Maybe there is already a precident for this type of stuff, but aren't there ways to start kibana where it won't have a stdout?

We should probably use `kbnServer.server.log()` or whatever so that it actually goes to the users log file.